scripts/functions.sh

```diff
@@ -8,6 +8,7 @@ function buildClient {
     cd client
 
     # Install all or missing dependencies.
+
```

Comment:
This new line only makes commit logs and git blame more noisy, without any other benefits. Please revert

Reply:
This was unintentionally, I had edited this part to use NPM instead of Yarn. In order to test if there was any problem with the yarn. Despite being faster, NPM is more reliable. Only I forgot to remove that line.